pkg/nb/types.go

```diff
@@ -578,8 +578,19 @@ type UpdateBucketClassParams struct {
 
 // BucketReplicationParams is the params of bucket_api.put_bucket_replication()
 type BucketReplicationParams struct {
```

Comment:
While creating an obc and passing a replication policy through the CLI, we are unmarshaling the json, seems to me that it would break with these changes, right? 
Can you share the input you pass to NooBaa?

Reply:
if I'll insert `log_replication_info` inside `replication_policy` (as @dannyzaken suggested in https://github.com/noobaa/noobaa-core/pull/6937#discussion_r846781560) then it will resolve the unmarshaling issue, WDYT?

Reply:
yes, I think it'll work :)